packages/core-utils/src/__tests__/__snapshots__/route.js.snap

```diff
@@ -135,6 +156,27 @@ Array [
 ]
 `;
 
+exports[`util > route routeComparator should sort routes based off of transitOperators sort value when a match is found 1`] = `
+Array [
+  Object {
+    "agencyId": "abc",
+    "agencyName": "Agency 1",
+    "longName": "Intercity Train 2",
+    "mode": "RAIL",
+    "routeId": "1:it2",
+    "shortName": "IC",
+  },
+  Object {
+    "agencyId": "abc",
+    "agencyName": "Acclaimed agency 1",
+    "longName": "Express route X",
+    "mode": "BUS",
+    "routeId": "2:erx",
```

Comment:
Is this test working as expected? It seems like route `2:erx` should come before route `1:it2`?

Reply:
Yes, it is working as expected. In this test method, the route comparator is sent in a config that says that feed_id 2 should be prioritized over feed_id 1. https://github.com/ibi-group/otp-ui/blob/a87a4e26bc4d961d787207ff3f850b44929b6b77/packages/core-utils/src/__tests__/route.js#L71-L78